values/prometheus-operator/service-monitors.yaml

```diff
@@ -1,6 +1,4 @@
 - name: blackbox
-  additionalLabels:
-    prometheus: po-prometheus
```

Comment:
It needs the right prometheus to report to. Please set to the one in use

Reply:
hmm but it interferes with commonLabels the also define `prometheus` key ...